modules/core/src/test/java/org/apache/ignite/internal/processors/cache/binary/BinaryMetadataUpdatesFlowTest.java

```diff
@@ -209,10 +197,7 @@ public void testFlowNoConflictsWithClients() throws Exception {
             return;
 
         discoveryHook = new DiscoveryHook() {
-            @Override public void handleDiscoveryMessage(DiscoverySpiCustomMessage msg) {
-                DiscoveryCustomMessage customMsg = msg == null ? null
```

Comment:
Why this deletion?

Reply:
This logic was moved to DiscoveryHook class  to avoid duplication.